docs/conf.py

```diff
@@ -80,3 +81,14 @@
 html_theme_options = {
     "logo_only": True,
 }
+
+# See:
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "numpy": ("https://docs.scipy.org/doc/numpy/", None),
+    "sklearn": ("https://scikit-learn.org/stable/", None),
+    "pandas": ("https://pandas.pydata.org/docs/", None),
+    "joblib": ("https://joblib.readthedocs.io/en/latest/", None),
+    "huggingface_hub": ("https://huggingface.co/docs/huggingface_hub/main/en", None),
```

Comment:
does this actually work?

Reply:
Yes, it works indeed :)

See [here](https://skops.readthedocs.io/en/latest/modules/classes.html#skops.hub_utils.download), at the bottom the link to `huggingface_hub.snapshot_download`